packages/webapp/src/routes/+layout.ts

```
@@ -1,30 +1,40 @@
 export const load: LayoutLoad<LayoutData> = async ({ fetch }) => {
-	let settingsJson: ConfigSource | undefined;
+	let config: Config;
 	let errorMessage: string | undefined;
 
 	try {
 		const response = await fetch(
-			'https://raw.githubusercontent.com/rainlanguage/rain.strategies/ff343647234ed635eb0b4b0c1862cf78665cc5ec/settings.json'
+			'https://raw.githubusercontent.com/rainlanguage/rain.strategies/c300d06cc8b460b06104849679284a2b0600240d/settings.yaml'
 		);
 		if (!response.ok) {
 			throw new Error('Error status: ' + response.status.toString());
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)